to.etc.domui/src/main/java/to/etc/domui/dom/html/NodeBase.java

```diff
@@ -307,6 +309,7 @@ public void internalOnClicked(@NonNull ClickInfo cli) throws Exception {
 	final protected void changed() {
 		if(m_disableChanged > 0)
 			return;
+		setRenderStyles();
```

Comment:
This will again always render the styles. This method should not be called here.